packages/workflow/test/Helpers.ts

```diff
@@ -96,7 +93,8 @@ class NodeTypesClass implements INodeTypes {
 		},
 	};
 
-	async init(nodeTypes: INodeTypeData): Promise<void> { }
+	// eslint-disable-next-line
+	async init(nodeTypes: INodeTypeData): Promise<void> {}
```

Comment:
Assuming this is a stub for a future implementation? Otherwise not quite clear on why there is an empty async method in here.